### PR TITLE
Adding cut-n-paste installation instructions for MacOS and Ubuntu 

### PR DESCRIPTION
**Pull request by dchakrav-github**
- Installation from pre-built binaries
- Moved installation via source through cargo install post cut-n-paste instructions

*Issue #, if available:*
#131 

*Description of changes:*
Providing an easy cut-n-paste into terminal instructions for installing guard binary from pre-built binaries. Tested on Ubuntu and MacOS.

### CODE REVIEW

**Review by PatMyron — DISMISSED** [review]
Running these commands again for new versions is still going to have the oldest version first in [`PATH`](https://en.wikipedia.org/wiki/PATH_(variable)):
`echo "Set PATH to PATH=${PATH}/:~/.guard/$version/"`

File: README.md [hunk 1]
```diff
@@ -149,6 +149,31 @@ These tenets help guide the development of the Guard DSL:
...
+#### Installation from Pre-Built Release Binaries
+
+##### MacOS
+
+By default this is builds for macOS-10 (Catalina). It has been tested to work on macOS-11 (BigSpur). See [OS Matix](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#github-hosted-runners)
+
+1. Open terminal of your choice. Default `Cmd+Space`, type `terminal`
+2. Cut-n-paste the commands below (change version=X for other versions)
+```bash
+     curl -fsSLI -o /dev/null \
```
Comment (PatMyron) [inline README.md:161]: Could just not specify an output file for these instead, right?
```suggestion
     curl -fsSLI \
```

File: README.md [hunk 2]
```diff
@@ -149,6 +149,31 @@ These tenets help guide the development of the Guard DSL:
...
+       https://raw.githubusercontent.com/aws-cloudformation/cloudformation-guard/main/install-guard.sh | sh
+```
+Remember to add `~/.guard/bin/` to your `$PATH`.
+
+##### Ubuntu
+
+1. Open any terminal of your choice
+2. Cut-n-paste the commands below (change version=X for other versions)
+```bash
+     curl -fsSLI -o /dev/null \
```
Comment (PatMyron) [inline README.md:171]:
```suggestion
     curl -fsSLI \
```

File: install-guard.sh [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            get_os_type
+            wget https://github.com/aws-cloudformation/cloudformation-guard/releases/download/$VERSION/cfn-guard-v$MAJOR_VER-$OS_TYPE-latest.tar.gz -O /tmp/guard.tar.gz ||
+                err "unable to download https://github.com/aws-cloudformation/cloudformation-guard/releases/download/$VERSION/cfn-guard-v$MAJOR_VER-$OS_TYPE-latest.tar.gz"
+            tar -C ~/.guard/$MAJOR_VER -xzf /tmp/guard.tar.gz ||
+                err "unable to untar /tmp/guard.tar.gz"
+            ln -sf ~/.guard/$MAJOR_VER/cfn-guard-v$MAJOR_VER-$OS_TYPE-latest/cfn-guard ~/.guard/bin ||
+                err "unable to symlink to ~/.guard/bin directory"
+            ~/.guard/bin/cfn-guard help ||
+                err "cfn-guard was not installed properly"
+            echo "Remeber to SET PATH include PATH=${PATH}:~/.guard/bin"
```
Comment (PatMyron) [inline install-guard.sh:31]:
```suggestion
            echo "Remember to SET PATH include PATH=${PATH}:~/.guard/bin"
```

File: install-guard.sh [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+        *)
+            err "unsupported OS type $_ostype"
+            ;;
+    esac
+}
+
+
+get_latest_release() {
+    curl -fsSLI -o /dev/null -w %{url_effective} \
```
Comment (PatMyron) [inline install-guard.sh:56]:
```suggestion
    curl -fsSLI -w %{url_effective} \
```

Comment (dchakrav-github) [reply install-guard.sh:56]: I am suppressing the output for this curl to prevent unnecessary messages on stdout. Any reason you want that removed?

**Review by gandhek — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dchakrav-github) [reply README.md]: @PatMyron 
-I == --head (this is HEAD request)
-L == Location header (which is what the redirect 302 response provides)
These commands worked on both Ubuntu and MacOS that others tested with. Can you please update with what errors you ran into? 

man curl

Comment (PatMyron) [reply README.md]: doesn't make sense to pipe head request into being executed, but looks like both `-I` and `-o` got dropped from the README in the last force push 👍

**Review by omkhegde — APPROVED** [review]

**Pull request merged**